proto/qgb/query.proto

```diff
@@ -3,13 +3,44 @@ package qgb;
 
 import "google/api/annotations.proto";
 import "cosmos/base/query/v1beta1/pagination.proto";
+import "qgb/msgs.proto";
 // this line is used by starport scaffolding # 1
 
 option go_package = "github.com/celestiaorg/celestia-app/x/qgb/types";
 
 // Query defines the gRPC querier service.
 service Query {
+  // ValsetConfirm Queries ValSetConfirm
+  rpc ValsetConfirm(QueryValsetConfirmRequest) returns (QueryValsetConfirmResponse) {
+    option (google.api.http).get = "/gravity/v1beta/valset/confirm";
```

Comment:
I think it makes sense to rename this http endpoint to qgb instead of gravity too

Reply:
my bad, will fix